CLI/src/config/skeletons.xml

```diff
@@ -109,21 +109,22 @@
     <patterns>
       <pattern name="POINTER.INCREMENT" location="NODE" action="replace">
         @{
-          var item = "";
-          if (@Model.needCompute)
-          {
-            foreach (var receiver in @Model.displayableReceivers)
-            {
-              item += "COMPUTE " + receiver + " = " + receiver + (@Model.incrementDirection == "Down" ? " - " : " + ") + @Model.sender.ToString() + ".\n";
-            }
-          }
-          else
-          {
-            item += "ADD " + (@Model.incrementDirection == "Down" ? "-" : "") + @Model.sender.ToString() + " to ";
-            foreach (var receiver in @Model.displayableReceivers)
-              item += receiver + ", ";
-          }
-          item = item.Remove(item.Length - 2);
+        var item = "";
+        if (@Model.needCompute)
+        {
+        foreach (var receiver in @Model.displayableReceivers)
+        {
+        item += "COMPUTE " + receiver + " = " + receiver + (@Model.incrementDirection == "Down" ? " - " : " + ") + @Model.sender.ToString() + ".\n";
+        }
+        }
+        else
+        {
+        item += (@Model.incrementDirection == "Up" ? "ADD " : "SUBTRACT ") + @Model.sender.ToString();
+        item += (@Model.incrementDirection == "Up" ? " to " : " from ") ;
+        foreach (var receiver in @Model.displayableReceivers)
+        item += receiver + ", ";
+        }
+        item = item.Remove(item.Length - 2);
```

Comment:
Indentation is used to have a somewhat more readable generated .cs file, so it would be better if you preserve the original style.
Also, here you can write C# code so you can factorize the test on the increment direction like this :
```
...
else
{
    if (@Model.incrementDirection == "Up")
    {
        item += "ADD " + @Model.sender.ToString() + " to ";
    }
    else
    {
        item += "SUBTRACT " + @Model.sender.ToString() + " from ";
    }
    foreach 
    ...
```

Reply:
Fix indentation
https://github.com/TypeCobolTeam/TypeCobol/pull/1564/commits/af7ec2ab236ede632a9fdbfddf7fc7c26735d840